Pull MATE 1.18.x updates from kenmays/oi-userland

kenmays opened the pull request:
Pull 2017-05-08

kenmays committed:
Merge pull request #1 from OpenIndiana/oi/hipster

Pull 2017-05-0

alarcher commented:
Caja 1.16.5 requires mate-desktop-2.0 >= 1.16.2 so the build will fail on the server if the PR is merged.
It should be put in a subsequent PR.

kenmays commented:
Mate-desktop is defined for version 1.16.2.https://github.com/kenmays/oi-userland/blob/oi/hipster/components/desktop/mate/mate-desktop/Makefile


On Monday, May 8, 2017, 5:53:44 PM PDT, Aurélien Larcher <notifications@github.com> wrote:
Caja 1.16.5 requires mate-desktop-2.0 >= 1.16.2 so the build will fail on the server if the PR is merged.
It should be put in a subsequent PR.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

alarcher commented:
Exactly, as soon as the PR is merged caja will be built in an environment where mate-desktop 1.16.1 only is installed so it will fail:
- a first PR should include mate-desktop 1.16.2 so that it is built then installed on the build server,
- then a second PR will update caja which can now be built in the environment containing newly installed mate-desktop.
Keep it mind that the components are only installed after all of them have been published.
Thank you :)

alarcher reviewed (changes requested):
Build done and tested.
Merge commit should be removed and Caja should pe put in a second PR.

kenmays commented:
Closing this pull request for merge corrections.

alarcher commented:
@kenmays Keep it open, it is fine.

alarcher commented:
@kenmays Do like this:
git checkout oi/hipster
git checkout -b caja
to keep your caja commit in a branch.

git checkout oi/hipster
git rebase -i HEAD~9

then remove the lines with the merge commit and caja

git push -f 

to push again you branch, the PR will be updated automatically.

kenmays committed:
Revised for PR
	modified:   components/desktop/mate/caja/Makefile
	modified:   components/desktop/mate/mate-control-center/Makefile
	modified:   components/desktop/mate/mate-desktop/Makefile
	modified:   components/desktop/mate/mate-icon-theme/Makefile
	modified:   components/desktop/mate/mate-panel/Makefile
	modified:   components/desktop/mate/mate-terminal/Makefile
	modified:   components/desktop/mate/mate-themes/Makefile
	modified:   components/desktop/mate/mate-utils/Makefile

pyhalov commented on components/desktop/mate/mate-terminal/Makefile:
```diff
@@ -10,19 +10,20 @@
 COMPONENT_NAME=		mate-terminal
-COMPONENT_VERSION=      1.16.1
+COMPONENT_VERSION=      1.16.3
 COMPONENT_REVISION=	1
```
Please, remove COMPONENT_REVISION while updating COMPONENT_VERSION

pyhalov commented on components/desktop/mate/mate-control-center/Makefile:
```diff
@@ -15,7 +15,7 @@
 
 COMPONENT_NAME=		mate-control-center
-COMPONENT_VERSION=	1.16.0
+COMPONENT_VERSION=	1.16.1
```
COMPONENT_ARCHIVE_HASH is not updated

pyhalov commented on components/desktop/mate/mate-themes/Makefile:
```diff
@@ -16,13 +16,13 @@
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:c2909366b83e9e56e630fd5f51c5cf2f6993e0eb337717cac2663c708577606e
+    sha256:0fad212926a59aedbeaf45d5e96a2337cebd54cc17ced38f5e50ce46ce786885 
 COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/themes/3.20/$(COMPONENT_ARCHIVE)
```
Doesn't publish:
`` Couldn't find 'usr/share/themes/Menta/gtk-3.0/assets/slider-hover.png' in any of the specified search directories:``
and so on. Have you updated manifests and sample-manifests?

kenmays committed:
Fix Revision.

kenmays committed:
Fix HASH

kenmays committed:
Bump to caja 1.16.6

kenmays committed:
Bump to mate-themes 3.20.20

kenmays committed:
Update mate-themes manifest.

kenmays committed:
Update manifest copyright

kenmays committed:
Update sample.manifest.p5m

kenmays committed:
Fix mode.

kenmays committed:
Fix manifest.

kenmays committed:
Fix manifest #2

kenmays committed:
Update to marco 1.16.1

kenmays committed:
Update to pluma 1.16.2

kenmays committed:
Merge MATE 1.18

kenmays committed:
Fix revision

kenmays committed:
Bump to eom 1.18.2

kenmays commented:
Latest updates are at MATE 1.18 (2017-05-15). The older ones are updated for the latest MATE 1.16 updates to fix previous pull request by Alex.

alarcher commented:
@kenmays A few missing build dependencies were fixed in this branch https://github.com/OpenIndiana/oi-userland/pull/3014 It would be good to add them.

alarcher commented:
@kenmays @pyhalov @xen0l @Toasterson Do we want 64bit builds only?

kenmays commented:
MATE 1.18.x 64-bit eventually..

pyhalov commented:
For binaries - likely

Toasterson commented:
I would also say for binaries yes and for libs keep 32bit for now.

kenmays committed:
Bump to engrampa 1.18.2

kenmays committed:
Bump to libweather 1.18.1

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64 bit v2

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64 bit v2

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit v2

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64 bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays committed:
Fix for 64-bit

kenmays commented:
I converted MATE 1.18 entirely over to 64-bit, You can review from this point.

kenmays committed:
Bump to Mesa 17.1.1

kenmays committed:
Bump to Mesa 17.1.1 v0.1

kenmays committed:
Bump to Mesa 17.1.1 v0.1a

alarcher commented:
@kenmays @pyhalov Thank you for your work on MATE! It looks all good!

pyhalov commented:
Superseded by #3177 , #3178, #3179 , #3181, #3182
